R/simmicmenten.R

```diff
@@ -0,0 +1,42 @@
+simmicmenten <- function(x, vmax, km, cv, reps, h = 1, log=F) {
```

Comment:
How about a different name?: simmm, simmicmen?

Reply:
simmicmen seems fine to me. simmm seems a little too vague.

Reply:
Honestly, I'm not quite sure how to do this. I could create a new roxygen document and all of that by generating a new use_r() call, but then I'm not quite sure how to properly get rid of the old function.

I'm going to leave it as is for now until I have some more time later to change the name to simmicmen